mindjunkies/courses/models.py

```
@@ -166,11 +172,24 @@ class Module(BaseModel):
 
     def __str__(self):
         return f"{self.title} - {self.course.title}"
+    
+
+    
+
+    def clean(self):
+        super().clean()
+        if hasattr(self, 'course') and self.course and self.order is not None:
+            if Module.objects.filter(course=self.course, order=self.order).exclude(pk=self.pk).exists():
+                raise ValidationError(f"Order {self.order} already exists in this Course.\nModule cannnot have same order")
```

Ruff (0.8.2)

Line too long (123 > 119)

(E501)